### PR TITLE
feat: updated get positions script and insolvency task

### PR DESCRIPTION
**Pull request by avniculae**

### CODE REVIEW

File: tasks/checkInsolvencyAtMaturity.ts [hunk 1]
```diff
@@ -6,28 +6,24 @@ import {
...
 ).setAction(async (taskArgs, hre) => {
-  // deploy FixedAndVariableMath as contract to be able to call function
-  const fixedAndVariableMathFactory = await hre.ethers.getContractFactory(
-    "FixedAndVariableMathTest"
-  );
-  const fixedAndVariableMath =
-    (await fixedAndVariableMathFactory.deploy()) as FixedAndVariableMathTest;
+  const fixedAndVariableMath = (await hre.ethers.getContractAt(
+    "FixedAndVariableMathTest",
+    "0x2D2EE238Ca74B546BfA64864f5654b5Ed7673f87"
```
Comment (CyclopsRex) [inline tasks/checkInsolvencyAtMaturity.ts:23]: I guess this still only works with one network? If we fell back to deploying a new FixedAndVariableMathTest as before, would that be enough to save gas on mainnet but to make this task portable across networks?

File: tasks/checkInsolvencyAtMaturity.ts [hunk 2]
```diff
@@ -41,7 +37,14 @@ task(
   );
 
   let noneInsolvent = true;
-  for (const marginEngineAddress of marginEngineAddresses) {
+  for (const poolName in poolAddresses) {
```
Comment (CyclopsRex) [inline tasks/checkInsolvencyAtMaturity.ts:40]: (Why) is this an improvement? Seems like the previous approach had an advantage of being network agnostic and also ignoring margin engines with no relevant positions?

File: scripts/getPositions.ts [hunk 3]
```diff
@@ -35,8 +38,8 @@ const getPositionsQueryString = `
 export async function getPositions(
   onlyActivePositions: boolean = false
 ): Promise<Position[]> {
-  const endpoint =
-    "https://api.thegraph.com/subgraphs/name/voltzprotocol/mainnet-v1";
+  const endpoint = process.env.SUBGRAPH_URL || "";
+  console.log(endpoint);
```
Comment (ssbarbee) [inline scripts/getPositions.ts:42]:
```suggestion

```

File: tasks/checkInsolvencyAtMaturity.ts (hunk 2 above)
Comment (avniculae) [reply tasks/checkInsolvencyAtMaturity.ts:40]: I agree, keeping it network agnostic is a good idea.

File: tasks/checkInsolvencyAtMaturity.ts (hunk 1 above)
Comment (avniculae) [reply tasks/checkInsolvencyAtMaturity.ts:23]: Good idea, thanks

**Review by 0xZenus — APPROVED** [review]

**Pull request merged**